README.md

```diff
@@ -16,19 +16,17 @@
 
 <br>
 
-ReactPHP is a low-level library for event-driven programming in PHP. At its core
-is an event loop, on top of which it provides low-level utilities, such as:
+ReactPHP is a low-level library for event-driven programming in PHP. The core of ReactPHP is an event loop built on top of the low-level utilities, such as:
```

Comment:
ReactPHP consists of multiple "core components": https://github.com/reactphp/react#core-components